nvflare/private/fed/server/fed_server.py

```diff
@@ -379,9 +379,7 @@ def GetTask(self, request, context):
 
                 current_model = shareable_to_modeldata(shareable, fl_ctx)
                 task.data.CopyFrom(current_model)
-                self.logger.info(f"Return task:{taskname} to client:{client.name} --- ({token}) ")
-
-                # self.fl_ctx.merge_sticky(fl_ctx)
+                self.logger.info(f"GetTask: Return task: {taskname} to client: {client.name} ({token}) ")
```

Comment:
I think we can add an if here: if the taskname is "__try_again__", then make the logger to debug, otherwise stay info. Similar handling can be applied to other cases

Reply:
So you suggest we move all logging related to the concept of "try_again" into debug level.

On one hand I feel that adding an "if" statement for the "try_again" is not that clean.

What are your thoughts @holgerroth @yanchengnv ?

Reply:
debug log level makes sense to me.